packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1073,4 +1074,12 @@ export default class SalesforceCRMService implements CRM {
     if (companyValue === onBookingWriteToRecordFields[companyFieldName]) return;
     return companyValue;
   }
+
+  private attendeeHasFreeEmailDomain(attendeeEmail: string) {
```

Comment:
Better name would be something like `shouldSkipAttendee` or `shouldConsiderAttendee` (and then use negation of it)

Right now it gives a sense that attendee has free email domain if it is true but it could be that he has free email domain but the organizer still wants him to be considered.